tests/x11/reboot_and_install.pm

```diff
@@ -15,14 +15,21 @@ use strict;
 
 use testapi;
 use utils qw(power_action workaround_type_encrypted_passphrase);
-use version_utils 'is_sle12_hdd_in_upgrade';
+use version_utils qw(is_sle12_hdd_in_upgrade sle_version_at_least);
 
 use bootloader_setup;
 use registration;
 
 sub run {
     # reboot from previously booted hdd to do pre check or change e.g. before upgrade
-    power_action('reboot');
+    if (sle_version_at_least('12')) {
+        power_action('reboot');
+    }
+    # backward support for SLES11, workaround for SLES11 power_action
+    else {
+        select_console 'root-console';
+        type_string "reboot\n";
+    }
```

Comment:
Seems the above change would affect opensuse test either, with the change above all opensuse tests on tw or leap 15 (with reboot_and_install.pm) will go to else condition, right?

Reply:
I will add opensuse tw or leap support here.

Reply:
Maybe a verification test run on tw is good here.

Reply:
Verification run on Opensuse
http://10.67.17.147/tests/2070#step/reboot_and_install/6

Reply:
@Jiawei517 No change for reboot_and_install.pm is required now, because we have switched to text mode reboot in sle upgrade test, see: https://openqa.suse.de/tests/1412220#step/reboot_to_upgrade/2

Reply:
Great, I will drop this.